datanode/gateway/graphql/schema.graphql

```diff
@@ -5127,10 +5129,8 @@ type ProposalDetail {
   state: ProposalState!
   "RFC3339Nano time and date when the proposal reached the Vega network"
   datetime: Timestamp!
-  "Terms of the proposal for proposal"
+  "Terms of the proposal"
   terms: ProposalTerms
-  "Terms of the proposal for a batch proposal"
-  batchTerms: BatchProposalTerms
```

Comment:
I don't think you can do this, because the batchTerms would be missing in another endpoint. This types is shared.